src/controllers/userProfileController.js

```diff
@@ -487,19 +487,19 @@ const userProfileController = function (UserProfile, Project) {
       }
       // validate userprofile pic
 
-      if (req.body.profilePic) {
-        const results = userHelper.validateProfilePic(req.body.profilePic);
+      // if (req.body.profilePic) {
+      //   const results = userHelper.validateProfilePic(req.body.profilePic);
 
-        if (!results.result) {
-          res.status(400).json(results.errors);
-          return;
-        }
-      }
+      //   if (!results.result) {
+      //     res.status(400).json(results.errors);
+      //     return;
+      //   }
+      // }
 
       const canEditTeamCode =
         req.body.requestor.role === 'Owner' ||
         req.body.requestor.role === 'Administrator' ||
-        req.body.requestor.permissions?.frontPermissions.includes('editTeamCode');
+        await hasPermission(req.body.requestor, 'editTeamCode');
```

Comment:
You can remove the owner and admin checks above this since hasPermission() checks role permissions.